### PR TITLE
Update focus area issue template for 2024.

### PR DESCRIPTION
**Pull request by jgraham**
This tries to explicitly collect more data than the template from 2023, but the majority of it remains optional. In particular it provides different spaces for different kinds of supporting data, so it's easier to see which proposals have which kinds of evidence backing up their likely impact.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/focus-area-proposal.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- type: textarea
+  attributes:
+    label: Standards Positions
+    description: |
+      Links to [Gecko](https://mozilla.github.io/standards-positions/) or [WebKit](https://webkit.org/standards-positions/) Standards Positions, or Intent to Prototype / Ship emails for the feature.
+- type: textarea
+  attributes:
+    label: Browser bug reports
+    description: |
+      Browser bugs or browser bug statistics, for example, [Chromium](https://bugs.chromium.org/), [Gecko](https://bugzilla.mozilla.org/),  or [WebKit](https://bugs.webkit.org/).
```
Comment (gsnedders) [inline .github/ISSUE_TEMPLATE/focus-area-proposal.yml:70]:
```suggestion
      Browser bugs or browser bug statistics, for example, [Chromium](https://bugs.chromium.org/), [Gecko](https://bugzilla.mozilla.org/), or [WebKit](https://bugs.webkit.org/).
```

File: .github/ISSUE_TEMPLATE/focus-area-proposal.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    label: Open Issues
+    description: |
+      Links to any open standards issues that affect the proposal.
+- type: textarea
+  attributes:
+    label: Tests
+    description: |
+      Tests for the feature in web-platform-tests on [wpt.fyi](https://wpt.fyi/).
+
+      These are the tests that will be used to score progress in this Focus Area. If the tests are not yet ready link to evidence of the tests being worked on (e.g. a PR in the web-platform-tests repository or an issue in a vendor bugtracker).
```
Comment (gsnedders) [inline .github/ISSUE_TEMPLATE/focus-area-proposal.yml:46]:
```suggestion
      These are the tests that will be used to score progress in this Focus Area. If the tests are not yet ready, link to evidence of the tests being worked on (e.g. a PR in the web-platform-tests repository or an issue in a vendor bug tracker).
```

**Review by gsnedders — COMMENTED** [review]
A few editorial comments to start with!

**Review by foolip — APPROVED** [review]
Approving but not landing until Sep 14.

**Pull request merged**